ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -18,6 +18,7 @@ aggregation:
   min: "Minimum value (e.g. Min_Ventilation_Air_Flow_Setpoint)"
   total: "Sum total of some set of values (e.g. total_request_heating_count)"
 component:
+  ballast: "An electrical ballast is a device placed in series with a load to limit the amount of current in an electrical circuit. Commonly a component of light fixtures."
```

Comment:
The term "ballast" typically refers only to fluorescent lighting, while the term driver is used for LEDs.
Here is some [reference](http://www.needhamelectric.com/ignites-lights-ballasts-vs-drivers/#:~:text=Fluorescent%20ballasts%20provide%20an%20initial,anode%20within%20the%20discharge%20tube.&text=LED%20drivers%20convert%20high%20voltage,are%20designed%20to%20run%20on.) for context.
A more generic term could be "control gear".
We should add both "ballast" and "driver".

Reply:
@pisuke thanks for the comment. "driver" is added